app/views/evaluator_submission_assignments/_submission_row.html.erb

```diff
@@ -1,25 +1,32 @@
-<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user">
+<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user" class="border-y-2px border-primary" style="height: 4rem;">
   <td data-label="Submission ID" class="text-top text-bold">
-    <% if assignment.status == "recused" %>
-      <%= image_tag('images/usa-icons/error.svg', class: "usa-icon--size-3", alt: "Recused", style: "vertical-align: middle; margin-right: 5px;") %>
-    <% end %>
-    <%= assignment.submission.id %>
+    <div class="margin-bottom-1 mobile-lg:margin-bottom-3">
+      <%= link_to submission_path(assignment.submission), class: "text-primary" do %>
+        <span class="mobile-lg:display-none">Submission ID </span><%= assignment.submission.id %>
+      <% end %>
+    </div>
```

Comment:
this text should be center-aligned. please ignore the figma on this, we don't need to add the margin-bottom here.
```suggestion
    <%= link_to submission_path(assignment.submission), class: "text-primary" do %>
      <span class="mobile-lg:display-none">Submission ID </span><%= assignment.submission.id %>
    <% end %>
```